frontend/device/gesturedetector.lua

```diff
@@ -556,16 +557,22 @@ function GestureDetector:handlePan(tev)
         end
 
         if msd_distance > self.MULTISWIPE_THRESHOLD then
+            local pan_ev_ms = pan_ev
```

Comment:
I just pondered for a few... thousands of milliiseconds :) with that `_ms` suffix - which looks too much like what we'd use to indicate milliseconds. Any other possible suffix? _msd _multiswipe?

Reply:
I could also just call it _copy, although that might be a bit misleading when it's not a copy.